packaging/Makefile

```diff
@@ -34,15 +35,18 @@ routesrv:
 	GO111MODULE=$(GO111) GOOS=$(GOOS) GOARCH=$(GOARCH) $(GOARM) CGO_ENABLED=$(CGO_ENABLED) go build -o routesrv ../cmd/routesrv/*.go
 
 clean:
-	rm -rf skipper eskip webhook routesrv build/
+	rm -rf $(BINARIES) build/
+
+docker.build: clean $(BINARIES)
```

Comment:
How does this relate to `docker-build` (with a dash) target?

Reply:
It came from this version: https://github.com/zalando/skipper/blob/41b621e7ea402c5f176456609e6c05d292bb20f8/packaging/Makefile#L40-L45 which is a bit inconsistent (see usage of build and buildx)

With this: `docker.build` is the "local" build (no architecture suffix, builds for local architecture) and uses the steps defined before.

Whereas `docker.build.amd64` etc. cross-compile for the target architectures using the steps at the bottom and buildx. `docker-build` and `docker-push` are wrappers to to invoke all three builds for the linux OS.

We could rename `docker.build` to `docker.build.local` or similar to avoid confusion.